dimagi_lake/migration/domain_migration.py

```diff
@@ -0,0 +1,51 @@
+from pyspark import SparkFiles
+
+import env_settings
+from dimagi_lake.utils import get_db_name
+from spark_session_handler import SPARK
+
+
+def drop_table_query(db_name, table_name):
+    return f"DROP TABLE IF EXISTS {db_name}.{table_name}"
+
+
+def create_table_query(db_name, table_name, data_path):
+    return f"CREATE TABLE {db_name}.{table_name} using delta LOCATION  '{data_path}';"
+
+
+def create_db(db_name):
+    databases = [database.name for database in SPARK.catalog.listDatabases()]
+    if db_name not in databases:
+        SPARK.sql(f"CREATE DATABASE {db_name}")
+        print(f"NEW DATABASE CREATED: {db_name}")
+
+
+def recreate_tables(db_name):
+
+    def _get_migration_queries(migration_config):
+        table_name = migration_config['table_name']
+        data_path = f"{env_settings.HQ_DATA_PATH}{migration_config['data_path']}"
+        return {
+            "table_name": table_name,
+            "drop_table": drop_table_query(db_name, table_name),
+            "create_table": create_table_query(db_name, table_name, data_path)
+        }
+
+    def _run_migration_df2(query):
```

Comment:
what does `df2` stand for and can this method name be improved?

Reply:
We can change the name of the method!